lib/stat_tracker.rb

```diff
@@ -24,15 +25,17 @@ def self.from_csv(locations)
 ##games methods for iteration 2
 
   def highest_total_score
-    games.max_by do |game|
-      self.calculate_total_score(game)
+    result = games.max_by do |game|
+      game.away_goals + game.home_goals
     end
+    self.calculate_total_score(result)
```

Comment:
I see what you're doing here now -- this should return an integer and not the actual game instnace. Thank you!